modules/tiled-gallery/tiled-gallery.php

```diff
@@ -103,7 +104,17 @@ public function gallery_shortcode( $val, $atts ) {
 
 		$this->set_atts( $atts );
 
-		$attachments = $this->get_attachments();
+		/**
+	         * Filters array of attachments for Tiled Gallery template.
+	         *
+	         * @module tiled-gallery
+	         *
+	         * @since 3.9.0
+	         *
+	         * @param array Array of attachments.
+	         * @param array Array of gallery shortcode attributes.
+	         */
+		$attachments = apply_filters( 'jetpack_gallery_attachments', array(), $this->atts );
```

Comment:
Somehow calling `apply_filters` here feels awkward to me. The `get_attachments` method disregards any incoming parameters, which is not how filters should work. What do you say we bring back the old code here and just filter the returned value in the `get_attachments` method? This way you will only need to change code in one place.

Thanks for contributing and please let me know if I can help with that.